api/src/main/java/io/smallrye/stork/api/observability/ObservationPoints.java

```diff
@@ -115,6 +115,10 @@ public int getDiscoveredInstancesCount() {
         public Throwable failure() {
             return failure;
         }
+
+      public long getSelectedInstance() {
```

Comment:
Maybe getSelectedInstanceId ? Otherwise I would have expected the ServiceInstance.